composer.json

```diff
@@ -16,16 +16,15 @@
   },
   "require-dev": {
     "alleyinteractive/alley-coding-standards": "^1.0",
-    "mantle-framework/testkit": "^0.7",
-    "phpunit/phpunit": "^9.3.3"
+    "mantle-framework/testkit": "^0.10"
   },
   "config": {
     "allow-plugins": {
       "dealerdirect/phpcodesniffer-composer-installer": true,
       "alleyinteractive/composer-wordpress-autoloader": true
     },
     "platform": {
-      "php": "7.4"
+      "php": "8.0.26"
```

Comment:
Why the specific version number? Could we use 8.0 instead?

Reply:
A number of packages actually require > 8.0.4

Reply:
(for my own education) what would happen if this was installed on a platform running 8.0.27? Is this considered a minimum required version?